.idea/jarRepositories.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you get rid of this file? Other than this, looks good!

Reply:
Oops yeah

Reply:
I'll finish working on fishing and I'll do that